geode-core/src/main/java/org/apache/geode/internal/cache/InternalCache.java

```diff
@@ -76,7 +76,9 @@
  */
 public interface InternalCache extends Cache, Extensible<Cache>, CacheTime {
 
-  InternalDistributedMember getMyId();
+  default InternalDistributedMember getMyId() {
```

Comment:
so the default implementation will result in a potential NPE?!?!? I think this should NOT have a default and each InternalCache should implement this.